ccd-definition/CaseField/CaseField-COS-nonprod.json

```diff
@@ -1,4 +1,53 @@
 [
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosNotifyClaimDetails1",
+    "Label": " ",
+    "FieldType": "NotifyClaimDetailsCoS",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosNotifyClaimDetails2",
+    "Label": " ",
+    "FieldType": "NotifyClaimDetailsCoS",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant1",
```

Comment:
Is it possible to use same label "cosLabelForClaimant1" for both places in notify claim and notify claim details? Two labels for showing same date.

Reply:
move to tab task